SetupCpp2015debug.bat

```diff
@@ -0,0 +1,8 @@
+mkdir build
```

Comment:
We need to remove these scripts. They aren't standardized yet and should be discussed how we want to do this.